scripts/make_COMPASS_ELM_forcing.py

```diff
@@ -1,247 +1,154 @@
+# SITE COORDS
+
+# Read in site/zone coordinate file
+# synoptic = \
+    (pd.read_csv('../data/raw/transect_coords/compass_synoptic.csv')
+    .assign(grid_points = lambda x: x.site_id + '_' + x.zone_id)
+    .assign(site_cat = 'synoptic')
+    .drop(columns=['distance'])
+     )
+
+# GET DISTANCE FROM NWI PERMANENT WATER BODY
+dist = pd.read_csv('../../data/site_pts/synoptic/synoptic_sites_pts_v2_wdist.csv')[['site_id', 'zone', 'distance']]
+dist.columns = ['site_name', 'zone_name', 'distance']
+
+# GET ELEVATION
+elev = pd.read_csv('../../output/results/elev/synoptic_elev_zone_v3.csv')[['site_name', 'zone_name', 'elev']]
+
+# Join elev and distance to synoptic file
+synoptic = \
+    (synoptic.merge(dist, on=['site_name', 'zone_name'], how='left')
+     .merge(elev, on=['site_name', 'zone_name'], how='left'))
+
+
+#%%--------------------------------------------------------------------------------------
+#  HYDRO
 # Hydrology uses the coastal wetland configuration setup for specifying a time series of hydrological boundary condition
-# Hourly time series, just do one year and the model will repeat it. Currently using water level of zero. 
-# Surface data set below will define ground surface height above drainage to give different hydrological conditions
-# Two grid cells (trough and high centered polygon)
-num_grids=len(grid_points)
-
-if __name__ == '__main__':
-    # These are different lengths. Erie starts in 1987 but data before that is a harmonic fit
-    # Anapolis starts in 1984
-    Erie_hydro=xarray.open_dataset('hydro_forcing/LakeErie_Gageheight_0salt.nc',decode_times=False)
-    # Annapolis hydro has a short linear gap filling spot in it that we might want to fix
-    Annapolis_hydro=xarray.open_dataset('hydro_forcing/Annapolis_schismPlus2_Peter_salinity_WT6_1_39yrs_NAVD.nc',decode_times=False)
-
-    ntimes=len(Annapolis_hydro['time'])
-    tide_data_multicell=xarray.Dataset(
-        data_vars={'tide_height':xarray.Variable(('time','gridcell'),data=np.zeros((ntimes,num_grids))+0.1,attrs={'units':'m'}),
-                'tide_salinity':xarray.Variable(('time','gridcell'),data=np.zeros((ntimes,num_grids)),attrs={'units':'ppt'}),
-                # Setting nonzero nitrate so leaching doesn't become a problem
-                'tide_nitrate':xarray.Variable(('time','gridcell'),data=np.zeros((ntimes,num_grids))+0.3e-3,attrs={'units':'mol/L'}),
-                },
-        coords   ={'time':xarray.Variable(('time',),data=np.arange(ntimes),attrs={'units':'hours'}),
-                    'gridcell':np.arange(num_grids)},
-        attrs    ={'Description':'Hydrological boundary conditions for grid cells'}
+# Hourly time series, just do one year and the model will repeat it. Currently using water level of zero.
+# These data were compiled, outlier filtered, gapfilled and cut to an identical record period across sites.
+hydro_filled_df = (
+    pd.read_csv('../../output/results/hydro_forcing_gauges/buoy_wl_all_syn_v4_filled_fixed.csv'))
```

Comment:
And I don't have this dataset

Reply:
Added data to:  **data/processed/synoptic_site_pts**  in the repo.